api/v2/perconaserverformysql_types.go

```diff
@@ -33,14 +32,14 @@ import (
 
 // PerconaServerForMySQLSpec defines the desired state of PerconaServerForMySQL
 type PerconaServerForMySQLSpec struct {
-	CRVersion             string    `json:"crVersion,omitempty"`
-	Pause                 bool      `json:"pause,omitempty"`
-	SecretsName           string    `json:"secretsName,omitempty"`
-	SSLSecretName         string    `json:"sslSecretName,omitempty"`
-	SSLInternalSecretName string    `json:"sslInternalSecretName,omitempty"`
-	MySQL                 MySQLSpec `json:"mysql,omitempty"`
-	Orchestrator          PodSpec   `json:"orchestrator,omitempty"`
-	PMM                   *PMMSpec  `json:"pmm,omitempty"`
+	CRVersion             string           `json:"crVersion,omitempty"`
+	Pause                 bool             `json:"pause,omitempty"`
+	SecretsName           string           `json:"secretsName,omitempty"`
+	SSLSecretName         string           `json:"sslSecretName,omitempty"`
+	SSLInternalSecretName string           `json:"sslInternalSecretName,omitempty"`
```

Comment:
We have the same fields in `PodSpec` struct. Should we keep them both? @delgod @hors

Currently we're using only these fields.

Reply:
1. I am not sure why we need fields in `PodSpec`. there are should be only one place for specifying certificates.
2. we should use internal certificates in `clone_ssl_*` options